assets/src/components/v2/cr_departures.tsx

```diff
@@ -1,20 +1,78 @@
 import BaseDepartureTime from "Components/eink/base_departure_time";
 import Arrow from "Components/solari/arrow";
-import React from "react";
+import useInterval from "Hooks/use_interval";
+import React, { useState } from "react";
 import { classWithModifier, imagePath } from "Util/util";
 import CRIcon from "./bundled_svg/cr_icon";
 import Free from "./bundled_svg/free";
 import ClockIcon from "./clock_icon";
 
+interface Departure {
+  arrow: string,
+  headsign: {
+    headsign: string,
+    variation: string
+  },
+  time: any,
+  track_number: number
+}
 interface CRDeparturesProps {
-  departures: any,
+  departures: Departure[],
   destination: string,
   time_to_destination: string,
   show_via_headsigns_message: boolean
 }
 
+const DeparturesTable: React.ComponentType<any> = (props) => {
+  const {departures} = props
+
+  let [headsignPageOne, setHeadsignPageOne] = useState(true);
+
+  useInterval(() => {
+    setHeadsignPageOne(!headsignPageOne);
+  }, 2000);
```

Comment:
[Checked with Mindy and co](https://mbta.slack.com/archives/C039ARUM48H/p1660939796076159) on this! Let's go with every 4 seconds.
```suggestion
  }, 4000);
```